releasenotes/notes/sampler_as_positional_arg_for_qnspsa-2efca170947f3cd7.yaml

```
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The list of arguments for `QNSPSA.get_fidelity()` has been updated to
+    include only the `ansatz` and the `sampler`, and removing previous
+    wildcard option. From now on, the sampler option must be explicitly
+    especified, as a positional or keyword argument, and it cannot be `None`.
```

```suggestion
    specified, as a positional or keyword argument, and it cannot be `None`.
```
Should fix the Checks spelling failure. In the other file the Checks are failing as the copyright year needs updating.
```
Wrong Copyright Year:'qiskit_algorithms/optimizers/qnspsa.py':  Current:'# (C) Copyright IBM 2021, 2024.' Correct:'# (C) Copyright IBM 2021, 2025.'

I had done another PR which fixed the other CI failures and having merged that and now updated this PR from main most is passing except the above 2 items. The CI failure against Qiskit Main is expected as there are a lot of changes needed to make it Qiskit 2.0 compatible, but those checks are not required to pass unlike the Checks which is required and is failing due to the above couple of items.
```

---

As noted above, in the latter part of that comment, the copyright date of that changed file qnspsa.py needs updating otherwise, as you can see from the CI that just ran with the one spelling change applied, its still failing,

---

Oh! Sorry about that...